ccds/__main__.py

```diff
@@ -31,6 +33,9 @@ def _main(*args, **kwargs):
         f.params[1].default = (
             "https://github.com/drivendataorg/cookiecutter-data-science"
         )
+        # The fifth parameter is the "checkout" option in the cookiecutter cli
+        # Per #389, set this to the currently released version by default
+        f.params[4].default = __version__
```

Comment:
I feel like first param being the repo is unlikely to change, but I am a little worried about ones later in the ordering. Is there a way we can loop over parameters and check by name to find the right one?